maps/_map.dm

```diff
@@ -154,22 +166,20 @@ proc/get_base_turf(var/z)
 	set desc = "Set the base turf for a z-level. Defaults to space, does not replace existing tiles."
 
 	if(check_rights(R_DEBUG, 0))
-
 		if(!holder)
 			return
-
 		var/choice = input("Which Z-level do you wish to set the base turf for?") as null|num
 		if(!choice)
 			return
-
 		var/new_base_path = input("Please select a turf path (cancel to reset to /turf/space).") as null|anything in typesof(/turf)
 		if(!new_base_path)
 			new_base_path = /turf/space //Only hardcode in the whole thing, feel free to change this if somewhere in the distant future spess is deprecated
-		for(var/obj/structure/docking_port/destination/D in all_docking_ports)
-			if(D.z == choice)
-				D.base_turf_type = new_base_path
-		var/datum/zLevel/L = map.zLevels[choice]
-		L.base_turf = new_base_path
+		var/update_old_base = alert(src, "Do you wish to update the old base? This will LAG.", "Update old turfs?", "Yes", "No")
+		update_old_base = update_old_base == "No" ? 0 : 1

```

Comment:
Might just be reading this wrong because its so late but isn't this the opposite